Soccar [Game-Server]/src/nl/soccar/gameserver/SessionController.java

```diff
@@ -25,7 +23,6 @@
 import nl.soccar.socnet.connection.Connection;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'nl.soccar.socnet.connection.Connection'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUselessImportCheck)